src/openforms/submissions/api/views.py

```diff
@@ -106,3 +109,32 @@ def get(self, request, *args, **kwargs):
     def perform_destroy(self, instance):
         remove_upload_from_session(instance, self.request.session)
         instance.delete()
+
+
+@extend_schema(
+    summary=_("View suspended submissions."),
+)
+class SuspendedSubmissionListView(ListAPIView):
+    serializer_class = SuspendedSubmissionSerializer
+    pagination_class = PageNumberPagination
+
+    queryset = Submission.objects.filter(
+        completed_on__isnull=True,
+        suspended_on__isnull=False,
+    )
+
+    # TODO authentication? permissions?
+    # TODO we need something here to verify current requests is allowed to access this BSN
+
+    def get_queryset(self):
+        qs = super().get_queryset()
+        bsn = self.request.query_params.get("bsn")
+        if not bsn:
+            return qs.none()
+        else:
+            qs = qs.filter(
+                auth_info__value=bsn,
+                auth_info__attribute=AuthAttribute.bsn,
+                auth_info__attribute_hashed=False,
+            )
```

Comment:
you need to compare/filter by hash since suspended submissions will have these values hashed.